test/e2e/framework/framework.go

```diff
@@ -409,6 +420,10 @@ func (f *Framework) CreateNamespace(baseName string, labels map[string]string) (
 	return ns, err
 }
 
+func (f *Framework) RecordFlakeIfError(err error, optionalDescription ...interface{}) {
```

Comment:
Could you add some comment to this method?

Reply:
Done